backend/src/api/blocks.ts

```diff
@@ -212,9 +212,11 @@ class Blocks {
         };
       }
 
-      const auditScore = await BlocksAuditsRepository.$getBlockAuditScore(block.id);
-      if (auditScore != null) {
-        blockExtended.extras.matchRate = auditScore.matchRate;
+      if (config.MEMPOOL.AUDIT) {
+        const auditScore = await BlocksAuditsRepository.$getBlockAuditScore(block.id);
+        if (auditScore != null) {
```

Comment:
It's better to use `if (!auditScore)` if you want to catch both undefined and null, otherwise use `!== null`

Reply:
sure, I tend to avoid coercing to boolean for null checks out of habit, because it's a footgun when working with strings and numbers

Reply:
going to merge this for now so we can proceed, feel free to open a PR to tweak this later